build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     ext.wordPressUtilsVersion = '2.2.0'
     ext.wordPressLoginVersion = '0.0.8'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.68.0-alpha3'
+    ext.gutenbergMobileVersion = '4362-4058d2e2df671d97461018173fa38538053a77b1'
```

Comment:
👋 @guarani, should we update [this to the latest commit for the](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4362/commits) `1.68.0` release branch of Gutenberg Mobile? Code-wise there should be no changes - just release notes were updated.